recipes/pyftpdlib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyftpdlib" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "88c1dc3daa8de9ff5b25fd57ae8cc4390b5d034f87f2396ebd93eeb8f6879972" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: release-{{ version }}.tar.gz
+  url: https://github.com/giampaolo/pyftpdlib/archive/release-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pyftpdlib = clcache:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
Looking at the `setup.py` file, there are two "extra" dependencies: 

- PyOpenSSL to support ftps. This library is available via the `default` channel, so it seems sensible to include it. 
- pysendfile to speed up uploads on posix for speeding up uploads for python <3.3. This is not on `defaults` or `conda-forge`, so I'm less inclined to add it at this point (but maybe it could be added in the feedstock once a conda-forge recipe is added).

Reply:
FWIW am adding `pysendfile` in PR ( https://github.com/conda-forge/staged-recipes/pull/7375 ).

They since support newer Python 3 versions.

Windows is still not supported. Though it does seem to be something they are interested in. They just need a bit of help. ( https://github.com/giampaolo/pysendfile/issues/30 )